source/CMakeLists.txt

```diff
@@ -69,10 +69,42 @@ if(USE_CUDA)
   )
 endif()
 
+if(USE_ROCM)
```

Comment:
Are these updates also necessary in Makefile.Objects?

Reply:
No, as far as I know, Makefile only supports the compilation of CPU version. It's very difficult to support other device version's compilation with Makefile.